script.js

```diff
@@ -0,0 +1,19 @@
+// defining a function which adds the "hidden" class to the target element
+// the class is defined elsewhere, in CSS
+function showNavModal() {
+    console.log("showsomething")
```

Comment:
we don't need the `console.log` calls anymore (and better to take out since users might see them)